### ISSUE TITLE
Add validation for empty name in edit profile.

### ISSUE THREAD

**Issue by aaryankotharii** [anitab-org/mentorship-ios#139]
**Describe the bug**
While editing the profile, if name field is emptied, the backend accepts an empty name but does not change it. Hence its better to disable `save` button in EditProfile View if name is empty.

**To Reproduce**
1. Go to Edit Profile View.
2. Empty Name textfield.
3. click save.
4. there will be a `-` for name in Profile View.

**Expected behavior**
`save` button in EditProfile View should be disabled if name is empty.

**Screenshots**
| updating empty Name  | Name is empty ( - ) | logout and login |
| ------------- | ------------- | ------------- |
| <img src = "https://user-images.githubusercontent.com/53724307/93707447-5592ed00-fb4c-11ea-863a-427a77341968.png" > | <img src = "https://user-images.githubusercontent.com/53724307/93707451-59bf0a80-fb4c-11ea-961f-2e4d987d68a2.png" > | <img src = "https://user-images.githubusercontent.com/53724307/93707452-5a57a100-fb4c-11ea-8a33-ac08747c07bb.png" >  |

**Simulator :**
 - Device: iPhone SE
 - iOS: 13.7
 - Xcode: 11.7

**Comment by ednofedulo** [comment]
can i work on this?

**Comment by ednofedulo** [comment]
Since this issue is labeled with Hacktoberfest label, this repository need to add hacktoberfest to its topics, otherwise the contributions made here would not be contabilized to the event

**Comment by yugantarjain** [comment]
> Since this issue is labeled with Hacktoberfest label, this repository need to add hacktoberfest to its topics, otherwise the contributions made here would not be contabilized to the event

You just made a PR, can you confirm if it updated your dashboard or not?

**Comment by ednofedulo** [comment]
Before you closed it, it appeared in the dashboard but with this status ![image](https://user-images.githubusercontent.com/8947399/95008239-74828c00-05ee-11eb-9077-4f253bee3e1e.jpeg)

**Comment by aaryankotharii** [comment]
Yes, `Hacktobfest` topic and `hacktoberfest-accepted` label is a requisite for pr to be counted.
here is the official [update](https://hacktoberfest.digitalocean.com/hacktoberfest-update)
<img width="1212" alt="Screenshot 2020-10-04 at 11 57 08 AM" src="https://user-images.githubusercontent.com/53724307/95008617-0c0ccc80-0639-11eb-9eac-620a243fdf0a.png">

**Comment by yugantarjain** [comment]
Got it, thanks!

**Comment by yugantarjain** [comment]
@ednofedulo I'm assigning it to you, thanks a lot for your interest in contributing to the Mentorship iOS project. For the Hacktoberfest, I'll confirm with the team if we're doing it or not.

A few suggestions for your next PR:
1. We don't want to disable the entire form, just the save button
2. Attach a screenshot to the PR if there are any visible changes
3. If you make a new method, add a unit test for that
4. A computed property may be better than a function for the disabled boolean

**Comment by yugantarjain** [comment]
@ednofedulo @aaryankotharii confirmed, Mentorship iOS is participating in Hacktoberfest! Thanks
